sass/layouts/slider.scss

```diff
@@ -0,0 +1,67 @@
+.slider__background {
+  background: $color-slider-bg;
+  border-bottom: 6px solid $color-slider-shadow;
+  position: relative;
+  top: 66px;
+  @include media-768 {
+    top: 47px;
+  }
+}
+
+.slider {
+  display: flex;
+  align-items: baseline;
+  max-width: 1021px;
+  margin: auto;
+  position: relative;
+  overflow: hidden;
+}
+
+.slider-line {
+  width: 2050px;
+  display: flex;
+  position: relative;
+  left: 0;
+  transition: all ease 1s;
+}
+
+#home {
+  position: relative;
+  width: 10px;
+  height: 10px;
+}
+
+.slider-controls {
+  position: absolute;
+  margin: 30% auto;
+  padding: 0 30px;
+  width: 100%;
+  height: auto;
+  z-index: 3;
```

Comment:
What is your rule for z-indexes? I have seen such values as 100 and 50 in your code, but here it is 3. It is better to select some step for z-indexes and set value according to your system.

Reply:
DONE!